smoke-tests/src/test/groovy/io/opentelemetry/smoketest/TomcatSmokeTest.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest
+
+import io.opentelemetry.proto.trace.v1.Span
+import okhttp3.Request
+
+class TomcatSmokeTest extends AppServerTest {
+
+  protected String getTargetImage(int jdk, String serverVersion) {
+    "ghcr.io/open-telemetry/java-test-containers:tomcat-${serverVersion}-jdk$jdk-20201207.405832649"
+  }
+
+  def "Server Handler test"() {
+    setup:
+    startTarget(8, "7.0.107")
```

Comment:
is this meant to be parameterized by the test params below?

Reply:
Quick answer - eventually yes, but not right now 😄 

Ideal solution eventually would be for the matrix to work as follows:

```
SmokeTestClass {
  forEach (configuration : matrix) {
    startTarget(configuration)
    testScenario1()
    testScenario2()
    ....
    tearDown(configuration)
 }
}
```

As a result there will be several separate test cases for each configuration in the matrix and the app server is started once per matrix entry.

Currently it is only possible to have either:

a) each `testScenarioX` will call `startTarget` which will effectively mean that each app server is started as many times as there are test cases. In case of this particular smoke test it would start 5 more tomcats.

or

b) There is one `testScenario` which contains all asserts in the world and it would be impossible to distinguish between failing tests.

The problem is I don't know yet how to achieve the ideal solution using neither Spock or JUnit.